### ISSUE TITLE
How do epochs work with ddpm?

### ISSUE THREAD

**Issue by sgbaird** [lucidrains/denoising-diffusion-pytorch#38]
For example, if I set 700k training steps, and stop at ~30k when there are 15k images, is there an idea of how many times the training data has been cycled through?

**Comment by XDUWQ** [comment]
The number of training is related to your data set. You may need to observe loss, FID and generation quality. In my personal experience, too many training times will lose diversity.
